### ISSUE TITLE
Language on the pages should be clearer

### ISSUE THREAD

**Issue by KarenSharp** [HTBox/crisischeckin#575]
It's not clear who "my" is, in "my commitments" or "my organization resources". Who is the website assuming that "I" am? Am "I" an admin? A volunteer? If I'm an individual person wanting to contribute, what does "my organization resources" mean? These two sections of the page should be on different pages, to disambiguate what action I am supposed to take. Every page should have only one obviously-apparent action I can take.

**Comment by NaveenP94** [comment]
Agreed, in order to simplify the UX we want to separate the features into many pages. Please see issue #500
